.travis.yml

```diff
@@ -5,6 +5,9 @@ php:
   - 5.6
   - 7
 
+before_script:
+  - phpenv config-rm xdebug.ini
```

Comment:
Xdebug is not stable yet on PHP7, and could lead to some PHP segfault errors.
As we dont use XDebug here, we disable it.